vendor/libavoid/mtst.cpp

```
@@ -0,0 +1,1094 @@
+    {
+        // Join this node to the previous node.
+        new HyperedgeTreeEdge(prevNode, node, nullptr);
+    }
+
+    return node;
+}
+
+void MinimumTerminalSpanningTree::buildHyperedgeTreeToRoot(VertInf *currVert,
+        HyperedgeTreeNode *prevNode, VertInf *prevVert, bool markEdges)
+{
+    if (prevNode->junction)
```

Cppcheck (2.20.0)

[error] Null pointer dereference

(ctunullpointer)